index.html

```diff
@@ -4,8 +4,39 @@
   <meta charset="utf-8">
   <meta name="Description" content="INF 7420 Project">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>Document</title>
+  <title>Julie Miller - Portfolio</title>
 </head>
 <body>
+  <header>
+    <nav>
+      <img src="images/kitty.jpg" alt="Julie Miller Logo">
+      <ul>
+        <li><a href="about-me.html">About Me</a></li>
+        <li><a href="contact.html">Contact</a></li>
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <div>
+      <img src="https://placekitten.com/200/300" alt="illustration of me as a librarian">
+      <figcaption>librarian & archivist</figcaption>
```

Comment:
if you want to use `<figcaption>` you have to have a `<figure>` element as the parent: 
```
<figure>
    <img src="/media/examples/elephant-660-480.jpg"
         alt="Elephant at sunset">
    <figcaption>An elephant at sunset</figcaption>
</figure>
```